github/provider_utils.go

```diff
@@ -0,0 +1,189 @@
+package github
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+	"testing"
+)
+
+var testUser string = os.Getenv("GITHUB_TEST_USER")
+var testCollaborator string = os.Getenv("GITHUB_TEST_COLLABORATOR")
+var isEnterprise string = os.Getenv("ENTERPRISE_ACCOUNT")
+var testOrganization string = os.Getenv("GITHUB_ORGANIZATION")
+var testOwner string = os.Getenv("GITHUB_OWNER")
+var testToken string = os.Getenv("GITHUB_TOKEN")
+
+var testTokenGHES string = os.Getenv("GHES_TOKEN")
+var testBaseURLGHES string = os.Getenv("GHES_BASE_URL")
+
+func testAccPreCheckEnvironment(t *testing.T, requiredEnvironmentVariables []string) {
+	for _, variable := range requiredEnvironmentVariables {
+		if v := os.Getenv(variable); v == "" {
+			t.Fatal(variable + " must be set for acceptance tests")
+		}
+	}
+}
+
+func testAccPreCheck(t *testing.T) {
+	if v := os.Getenv("GITHUB_TOKEN"); v == "" {
+		t.Fatal("GITHUB_TOKEN must be set for acceptance tests")
+	}
+	if v := os.Getenv("GITHUB_OWNER"); v == "" {
+		t.Fatal("GITHUB_OWNER must be set for acceptance tests")
```

Comment:
we should also check for GITHUB_ORGANIZATION here if owner not set

Reply:
pushed a commit to update the check to reflect this